lang/lang_main.php

```diff
@@ -9,30 +9,42 @@
  * @version 3.3.0
  */
 
+function load_lang_file(&$lang, $ui_lang, $filename, $parent_dir = ".") {
```

Comment:
I am not sure how to fix this. As explained, it's only present the files loaded with `require_once` can use `$lang` as global variable, instead of as local variable of the function.